src/ensembl/src/styles/_settings.scss

```diff
@@ -16,6 +16,9 @@ $global-radius: 0;
 $global-padding-left: 100px;
 $global-button-cursor: auto;
 
+// Image configurations
+$image_width: 695;
```

Comment:
Oh, that's far too global! I think we need to put it in some variables file inside the entity viewer folder. [This one](https://github.com/Ensembl/ensembl-client/tree/dev/src/ensembl/src/content/app/entity-viewer/gene-view/styles) looks appealing.

The name image_width is also not particularly informative. `gene_image_width`?